boba/bobarun.py

```diff
@@ -52,3 +54,12 @@ def run_commands_in_folder(folder, file_with_commands):
         for line in f.readlines():
             os.system(line)
     os.chdir(cwd)
+
+class PythonDict(click.Option):
```

Comment:
The name of the class is a bit too general. Maybe indicating it's a custom click command class. Also why not move it to `cli.py` (if it's only used there? I might be wrong)